drop-storage/src/types.rs

```diff
@@ -240,7 +249,10 @@ pub struct OutgoingPath {
     #[serde(serialize_with = "serialize_datetime")]
     pub created_at: NaiveDateTime,
     pub transfer_id: TransferId,
-    pub base_path: String,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub base_path: Option<PathBuf>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub content_uri: Option<url::Url>,
```

Comment:
This is a JSON API structure. The question is do we change something here regarding the URIs?
I see that even if we decide to not pass raw URIs here the `base_path` can be changed to `full_path` as it might be more useful for apps

Reply:
So we decided to not change the JSON API